api-docs/swagger-v1.1.yml

```diff
@@ -1108,10 +1108,14 @@ parameters:
     required: false
   personal_health_scores:
     name: personal_health_scores[]
-    type: string
+    type: array
+    items:
+      type: string
+      enum: ['A', 'B1', 'B2', 'C1', 'C2']
     in: query
     description: restrict by health score, repeat parameter
     required: false
+    collectionFormat: multi
```

Comment:
I've tested it locally, it works. Not sure about the look though.

![screen shot 2017-03-06 at 14 22 13](https://cloud.githubusercontent.com/assets/445211/23611607/573f1726-0278-11e7-876e-1731ac9e122e.png)